src/parsers/uniqueTokens.js

```diff
@@ -110,10 +109,11 @@ export const parseAccountUniqueTokens = data => {
             'twitter_username',
             'wiki_link',
           ]),
-          currentPrice: asset.sell_orders
+          currentPrice: asset.seaport_sell_orders
             ? `${
-                Number(asset.sell_orders[0].current_price) / 1000000000000000000
-              } ${asset.sell_orders[0].payment_token_contract.symbol}`
+                Number(asset.seaport_sell_orders[0].current_price) /
+                1000000000000000000
+              } ${asset.seaport_sell_orders[0].payment_token_contract.symbol}`
```

Comment:
This currently doesn't work as we `seaport_sell_orders` doesnt contain the symbol anywhere.